src/Solver.cpp.Rt

```diff
@@ -12,8 +12,8 @@
 #endif
 #include "cross.h"
 #include "Region.h"
-//#include "LatticeContainer.h"
-class LatticeContainer;
+//#include "CartLatticeContainer.h"
+class CartLatticeContainer;
```

Comment:
I think this is obsolete

Reply:
Closed in [066f2e7](https://github.com/CFD-GO/TCLB/pull/454/commits/066f2e7f930cf11249a8fcb6006ce47972e0dcdc)